scripts/install-deps-ubuntu.sh

```
@@ -22,77 +24,90 @@ sudo apt-get update && apt-get install -y \
     python-is-python3 \
     wget
 
-sudo apt update && apt install -y wget && \
-ARCH=$(dpkg --print-architecture) && \
+sudo apt update && sudo apt install -y wget
+ARCH=$(dpkg --print-architecture)
 case ${ARCH} in \
     amd64) GOARCH=amd64 ;; \
     arm64) GOARCH=arm64 ;; \
     *) echo "Unsupported architecture: ${ARCH}" && exit 1 ;; \
-esac && \
-wget https://go.dev/dl/go1.23.5.linux-${GOARCH}.tar.gz && \
-sudo rm -rf /usr/local/go && \
-sudo tar -C /usr/local -xzf go1.23.5.linux-${GOARCH}.tar.gz && \
+esac
+wget https://go.dev/dl/go1.23.5.linux-${GOARCH}.tar.gz
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf go1.23.5.linux-${GOARCH}.tar.gz
 rm go1.23.5.linux-${GOARCH}.tar.gz
 
-git clone https://github.com/flintlib/flint.git && \
```

Removed command chaining with `&& \` as it masks failures